### ISSUE TITLE
Pregain question

### ISSUE THREAD

**Issue by micahvdm** [AidaDSP/aidadsp-lv2#10]
So I just tested the latest version built via mpb and it’s really sounding great! A lot of very welcome changes!
The one thing I am noticing though is that the pregain know has little to no effect on pretty much all of your models. Is this a bug or is it meant to be super subtle?
I’m assuming the pregain would be used to reduce the amount of gain in the initial stages of the net, or am I wrong?

**Comment by spunktsch** [comment]
it works differently. The models are snapshots atm with all parameters at fixed values. 
The pregain only effects the input to the net - its more or less a boost function.

**Comment by micahvdm** [comment]
So currently there’s no ability to tone down the gain of a model?? This seems like a bit of a challenging way for the end user to operate. That would require having dozens of models for various gain settings…is the conditioned models only turned off in the ttl files?

**Comment by spunktsch** [comment]
Yeah this will be communicated that these are snapshots. The models in this repo are with the old training and dataset. 
The new one has far better response and dynamics. So the pregain has more of an effect. 
This is of course dependent of the initial amp. 

 Conditioned models are planned for a future release.

**Comment by micahvdm** [comment]
Ah rad, that makes sense! Any chance I can grab some of those new models to try? I’m busy setting up a training rig to capture some of my amps, can share them when done!

**Comment by spunktsch** [comment]
sure, no Problem. Do you have a specific amp in mind that you can compare it to or know by heart? Would be a good reference point for us to know what you think.

**Comment by micahvdm** [comment]
I have most of the NeuralDSP stuff, so I know their sounds, 5150II, VoxAC15, Fender Blues Jr, JCM 800 and a custom handwired 9W valve amp.

**Comment by spunktsch** [comment]
ok cool. Hit me up via mail and I'll send you one.

**Comment by micahvdm** [comment]
These new models are significantly better, in every way! Can you send me the process for training models please so I can start training some of my amps?

**Comment by MaxPayne86** [comment]
@micahvdm I will close the issue and we will continue discussion in pvt, thanks
